### ISSUE TITLE
css.types.image.gradient.conic-gradient - Does not work with Safari 12.1

### ISSUE THREAD

**Issue by PRR24** [mdn/browser-compat-data#25207]
### What type of issue is this?

Incorrect support data (example: BrowserX says "86" but support was added in "40")

### What information was incorrect, unhelpful, or incomplete?

Safari 12.1 does not support conic-gradient.

### What browsers does this problem apply to, if applicable?

Safari

### What did you expect to see?

conic-gradient to work according to support matrix

### Did you test this? If so, how?

The very first simple example of conic-gradient on MDN does not work.
https://css-gradient.com is not able to produce working example either.

### Do you have anything more you want to share?

Safari 14.1 works, don't have versions in between sadly.
linear- and radial-gradients work with Safari 12.1.

### MDN URL

https://developer.mozilla.org/en-US/docs/Web/CSS/gradient/conic-gradient

### MDN metadata

<details>
<summary>MDN page report details</summary>

* Query: `css.types.image.gradient.conic-gradient`
* Report started: 2024-11-25T15:45:44.240Z

</details>

**Comment by skyclouds2001** [comment]
note this will be moved to css.types.gradient.conic-gradient via https://github.com/mdn/browser-compat-data/pull/25191

**Comment by PRR24** [comment]
Unless it is already known, conic-gradient is supported since Safari 13.0 according to Safari release notes.

**Comment by yisibl** [comment]
> Unless it is already known, conic-gradient is supported since Safari 13.0 according to Safari release notes.

Yes, https://developer.apple.com/documentation/safari-release-notes/safari-13-release-notes#:~:text=Added%20support%20for%20CSS%20conic%20gradients
